### PR TITLE
Initial setup updates and fixes

### PR DESCRIPTION
**Pull request by shannonphu**
- Found some issues with using npm package debug and fixed
- Fixed issue with server and client using same port for me
- Updated to run entire app with concurrently so only 1 terminal needs to be run and logs are labelled and go to single terminal. This can be reverted back if you prefer separate terminals
- Added root level npm install script and root level run script
- Fixed issue with typo in file names in server router imports

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "install": "cd app; yarn install; cd ../server; yarn install",
     "server": "cd server; npm start",
     "client": "cd app; npm start",
-    "dev": "concurrently \"npm run server\" \"npm run client\"",
+    "dev": "concurrently -n server,client -c magenta,blue \"npm run server\" \"npm run client\"",
```
Comment (shivkumarganesh) [inline package.json:16]: This was a good add!

File: README.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ Simple integration with Twitter services in order to perform CRUD operations.
 ## Application setup
 
 1. Clone the code
+2. Don't check in your changes to the API keys. This will untrack the API key config file and allow you to keep your own version: 
+    ```
+    git update-index --skip-worktree server/config/twitter-config.js
```
Comment (shivkumarganesh) [inline README.md:9]: I will leave this update here but I already stopped tracking this file on .gitignore. This saves my keys. But good for people to know.

**Review by shivkumarganesh — APPROVED** [review]

**Comment by shivkumarganesh** [comment]
Checked the changes and commit.

File: README.md (hunk 2 above)
Comment (shannonphu) [reply README.md:9]: For some reason when I added my keys it still was tracking and I was also confused too, but it fixed.

**Pull request merged**